blog/2023-09-25-constructs-methods.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Designing programmatic APIs for cloud infrastructure using constructs (part 1)
```

Comment:
I am wondering if this should be called something like:

"Construct-based design pitfalls: Part 1 - Mutability"

Something like that.

Reply:
I totally forgot I was going to ask y'all for title suggestions - this is perfect 💯